adjust_dataset.py

```diff
@@ -0,0 +1,86 @@
+import pandas as pd
+import argparse
+
+HAPPINESS = 3
+
+# Alternative for users who don't pass arguments from command-line, that alternative will be deleted in future version
+path_input_file = "fer2013.csv"
+path_output_file = 'smile_warrior_dataset.csv'
+
+# Set that flag to 1 if you want to pass paths for csv files as a command-line arguments
+# Set that flag to 0 if you want to change paths for csv files by changing global variables above
+path_from_terminal = 1
+
+
+def main():
+    """
+    Run this script to create new csv file with dataset for Smile-Warrior from fer2013.csv dataset for Facial
+    Expression Recognition.
+
+    This script is changing labels of emotions in fer2013.csv to 1 - smile, 0 - no smile.
+
+    To run correctly you need to pass two paths to csv files as a command-line arguments, first for input file which
+    will be adjusted for Smile-Warrior needs and second for output file where new dataset will be created.
+
+    Example:
+
+        $ python adjust_dataset.py fer2013.csv smile_warrior_dataset.csv
+
+    """
+
+    if path_from_terminal == 0:
```

Comment:
This looks bad to give a default path to a file. Let's keep it simple. In README we have it as an argument anyway.

Reply:
Ok